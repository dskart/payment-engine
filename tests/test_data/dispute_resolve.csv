type,client,tx,amount
deposit,1,1,1.0
deposit,1,2,2.0
dispute,1,2,0.0
resolve,1,2,0.0
withdrawal,1,3,2.0
