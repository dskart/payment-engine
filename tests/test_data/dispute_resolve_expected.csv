client,available,held,total,locked
1,1.0,0.0,1.0,false
